examples/linear_model/plot_lasso_and_elasticnet.py

```diff
@@ -19,12 +19,11 @@
 # Generate some sparse data to play with
 np.random.seed(42)
 
-n_samples, n_features = 50, 200
+n_samples, n_features = 50, 100
 X = np.random.randn(n_samples, n_features)
-coef = 3 * np.random.randn(n_features)
-inds = np.arange(n_features)
-np.random.shuffle(inds)
-coef[inds[10:]] = 0  # sparsify coef
+idx = np.arange(n_features)
+coef = (-1) ** idx * np.exp(-idx / 10)
```

Comment:
I would add a comment explaining why you use this `coef`, along the lines of what you wrote in the PR
```
# using decreasing coefs with alternated signs for visualization purpose
```